### PR TITLE
Create media

### PR DESCRIPTION
**Pull request by jgondron**
Adds the Create action for media files. To create, perform a post with the required file params, ex:
```
{
  "file_path": "big_full.mov",
  "media_type": "video"
}
```
the response will be a serialized MediaFile, (note: this will change in DEC-1112) ex:
```
{
    "media_type": "video",
    "file_path": "big_full.mov",
    "uuid": "33b5e7f3-4b8b-46cf-b94f-de416b506f2a"
}
```
the response for validation errors will look like:
```
{
    "@context": "http://schema.org",
    "@type": "CreateAction",
    "actionStatus": "FailedActionStatus",
    "object": {
        "media_type": "",
        "file_path": "",
        "uuid": "8fbdcb50-fbb5-4a2c-9c32-7527ec365efb"
    },
    "error": [
        {
            "name": "file_path",
            "description": "can't be blank"
        },
        {
            "name": "media_type",
            "description": "can't be blank"
        }
    ]
}
```
Note: I sort of got guard working with rspec as part of this, but it restarts Spring every time a spec changes.

### CODE REVIEW

File: db/migrate/20160707201728_create_media_file.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class CreateMediaFile < ActiveRecord::Migration[5.0]
```
Comment (rfox2) [inline db/migrate/20160707201728_create_media_file.rb]: Just curious: why the 5.0 designation? I've not seen that before in a migration. That was probably generated for the migration but it's new to me.

**Comment by rfox2** [comment]
👍

**Comment by danthewolfe** [comment]
Should we add a status for success response?

File: db/migrate/20160707201728_create_media_file.rb (hunk 1 above)
Comment (jgondron) [inline db/migrate/20160707201728_create_media_file.rb]: Yeah, was generated. I wondered the same.

**Comment by jgondron** [comment]
Working on changing the error response

**Comment by danthewolfe** [comment]
👍

**Pull request merged**